test/base.py

```diff
@@ -1,12 +1,28 @@
 import unittest
-
 import jsonschema
+import os, sys
 
+sys.path.insert(1, os.path.join(sys.path[0], '..'))
+from genson import Schema
 
 class SchemaTestCase(unittest.TestCase):
+
+    def assertGenSchema(self, instance, options, expected):
+        actual = Schema(**options).add_object(instance).to_dict()
+        self.assertSchema(actual, expected)
+        self.assertValidData(instance, actual)
```

Comment:
Because test_gen_multi calls assertSchema, I don't believe the call signature can be changed.  The single-step "generate a schema from a single object, validate the schema, and validate the object" call is useful but doesn't supersede the standalone "validate the schema" call.